compiler/src/org.graalvm.compiler.nodes/src/org/graalvm/compiler/nodes/calc/BinaryArithmeticNode.java

```diff
@@ -261,7 +393,7 @@ private static ReassociateMatch findReassociate(BinaryNode binary, NodePredicate
      * @param forY
      * @param forX
      */
-    public static ValueNode reassociate(BinaryArithmeticNode<?> node, NodePredicate criterion, ValueNode forX, ValueNode forY, NodeView view) {
+    public static ValueNode reassociate1(BinaryArithmeticNode<?> node, NodePredicate criterion, ValueNode forX, ValueNode forY, NodeView view) {
```

Comment:
please give this method and the reassociate2 a better descriptive name that makes clear what is done in each method